webapp/src/pages/applicationForm/components/ApplicationForm.js

```diff
@@ -517,26 +531,46 @@ class Submitted extends React.Component {
           Your application will be reviewed by our committee and we will get back to you as soon as
           possible.
         </p>
+
         <p class="timestamp">
           You submitted your application on{" "}
           {this.props.timestamp && this.props.timestamp.toLocaleString()}
         </p>
+
         <div class="submitted-footer">
           <button class="btn btn-danger" onClick={this.handleWithdraw}>
             Withdraw Application
           </button>
         </div>
+
+        <div class="submitted-footer">
+          <button class="btn btn-danger" onClick={this.handleEdit}>
```

Comment:
Lets make this btn-warning perhaps, to better differentiate it from the Withdraw action

Reply:
<img width="578" alt="Screenshot 2020-03-24 at 09 27 15" src="https://user-images.githubusercontent.com/29726916/77399233-af8b4380-6db1-11ea-91ab-431ffb753b65.png">
this is what the change would look like. are you happy with it?
im pushing the change